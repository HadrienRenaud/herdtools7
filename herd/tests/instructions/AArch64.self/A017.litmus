AArch64 A017
Stable=X30
{
 (* necessary to generate an initial write for P0:L1 *)
 0:X1=P0:L1;
}
 P0           ;
 BL L0        ;
L1:           ;
 NOP          ;
L0:           ;
 LDR W0,[X30] ;
forall(0:X0=instr:"NOP")
