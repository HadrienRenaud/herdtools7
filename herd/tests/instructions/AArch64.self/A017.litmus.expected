Test A017 Required
States 1
0:X0=NOP;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=NOP)
Observation A017 Always 1 0
Hash=46b8ce018c9628c159332ad9f5900cf9

